Documentation/Storage-Configuration/Object-Storage-RGW/ceph-object-swift.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Object Store with Keystone and Swift
+---
+
+You can configure the Swift API and the Keystone integration of Ceph RGW natively via the Object Store CRD, which allows native integration of the Rook operated Ceph RGW into [OpenStack](https://www.openstack.org/) clouds.
+
+## Create a Local Object Store with Keystone and Swift
+
+This example will create a `CephObjectStore` that start the RGW service in the cluster providing a Swift API.
+Using Swift requires the use of [OpenStack Keystone](https://docs.openstack.org/keystone/latest/) as an authentication provider.
+
+!!! note
+This sample requires *at least 3 bluestore OSDs*, with each OSD located on a *different node*.
+
+The OSDs must be located on different nodes, because the [`failureDomain`](../../CRDs/Block-Storage/ceph-block-pool-crd.md#spec) is set to `host` and the `erasureCoded` chunk settings require at least 3 different OSDs (2 `dataChunks` + 1 `codingChunks`).
+
+!!! note
+This example assumes that there already is an existing OpenStack Keystone instance ready to use for authentication.
```

Comment:
Probably best to include this in the same note which will be moved below, to just above the yaml example.